gusto_embedded/README.md

```diff
@@ -94,29 +94,92 @@ yarn add @tanstack/react-query react react-dom
 For supported JavaScript runtimes, please consult [RUNTIMES.md](RUNTIMES.md).
 <!-- End Requirements [requirements] -->
 
-<!-- Start SDK Example Usage [usage] -->
 ## SDK Example Usage
 
 ### Example
+Automatic token refresh using a persistent data store.
 
 ```typescript
 import { GustoEmbedded } from "@gusto/embedded-api";
+import { CompanyAuthenticatedClient } = "@gusto/embedded-api/CompanyAuthenticatedClient";
 
-const gustoEmbedded = new GustoEmbedded({
-  companyAccessAuth: process.env["GUSTOEMBEDDED_COMPANY_ACCESS_AUTH"] ?? "",
-});
+class PersistentTokenStore implements TokenStore {
+  constructor() {}
+
+  async get() {
+    const { token, expires, refreshToken } = retrieveToken();
+
+    return {
+      token,
+      expires,
+      refreshToken,
+    };
+  }
+
+  async set({
+    token,
+    expires,
+    refreshToken,
+  }: {
+    token: string;
+    expires: number;
+    refreshToken: string;
+  }) {
+    saveToken(token, refreshToken, expires);
+  }
+}
+
+const client = new GustoEmbedded();
+const clientId = process.env["GUSTOEMBEDDED_CLIENT_ID"]
```

Comment:
If this module gets imported on the client, `process.env` will likely not be defined. What if we deferred this until a particular action is taken instead of making it happen in the module import?